packages/lesswrong/components/ultraFeed/UltraFeedItemFooter.tsx

```diff
@@ -83,6 +97,34 @@ const styles = defineStyles("UltraFeedItemFooter", (theme: ThemeType) => ({
     color: `${theme.palette.ultraFeed.dim} !important`,
     marginLeft: -8 // necessary to counter baked-in left margin from AgreementVoteAxis.tsx
   },
+  // Styles to override OverallVoteAxis children, applied only on small screens
+  footerVoteScoreOverride: {
+    // Default style for this class (applies on large screens)
+    fontSize: `${theme.typography.body2.fontSize}px !important`, 
+    margin: '0 7px !important', // Use default small margin for large screens
+
+    // Override for small screens
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '1.3rem !important',
+      margin: '0 7px !important',
+    }
+  },
+  hideSecondaryScoreOnMobile: {
+    [theme.breakpoints.down('sm')]: {
+      display: 'none !important',
+    }
+  },
+  // Styles to override AgreementVoteAxis score when used in footer
+  footerAgreementScoreOverride: {
+    // Default style for this class (applies on large screens)
+    fontSize: `${theme.typography.body2.fontSize}px !important`,
+    margin: '0 7px !important',
+    // Override for small screens
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '1.3rem !important',
+      margin: '0 7px !important', // Keep same margin for small screens based on user change above
```

Comment:
I really don't like all the `!important`

Reply:
yeah, me neither